backend/entityservice/tasks/comparing.py

```diff
@@ -52,33 +53,30 @@ def create_comparison_jobs(project_id, run_id, parent_span=None):
             log.warning("Unexpected number of dataset sizes in db. Stopping")
             update_run_mark_failure(conn, run_id)
             return
-        else:
-            lenf1, lenf2 = dataset_sizes
 
         encoding_size = get_project_encoding_size(conn, project_id)
 
-        size = lenf1 * lenf2
-
-        log.info("Computing similarity for {} x {} entities".format(lenf1, lenf2))
+        log.info(f"Computing similarity for "
+                 f"{' x '.join(map(str, dataset_sizes))} entities")
         current_span.log_kv({"event": 'get-dataset-sizes'})
 
-        filters1_object_filename = get_filter_metadata(conn, dp_ids[0])
-        filters2_object_filename = get_filter_metadata(conn, dp_ids[1])
+        filters_object_filenames = tuple(
+            get_filter_metadata(conn, dp_id) for dp_id in dp_ids)
         current_span.log_kv({"event": 'get-metadata'})
 
         log.debug("Chunking computation task")
 
     chunk_infos = tuple(anonlink.concurrency.split_to_chunks(
         Config.CHUNK_SIZE_AIM,
-        dataset_sizes=(lenf1, lenf2)))
+        dataset_sizes=dataset_sizes))
 
     # Save filenames with chunk information.
     for chunk_info in chunk_infos:
         chunk_dp1_info, chunk_dp2_info = chunk_info
-        assert chunk_dp1_info['datasetIndex'] == 0
-        chunk_dp1_info['storeFilename'] = filters1_object_filename
-        assert chunk_dp2_info['datasetIndex'] == 1
-        chunk_dp2_info['storeFilename'] = filters2_object_filename
+        chunk_dp1_info['storeFilename'] \
+            = filters_object_filenames[chunk_dp1_info['datasetIndex']]
+        chunk_dp2_info['storeFilename'] \
+            = filters_object_filenames[chunk_dp2_info['datasetIndex']]
```

Comment:
To avoid confusion in the future these shouldn't be called `chunk_dp1...` and `chunk_dp2...` anymore.

nit: not a fan of splitting simple assignments over two lines just to stay under a max line length.

Reply:
I’ve changed it to something else. Please let me know if you think it’s better.